core/src/main/java/com/linecorp/armeria/server/streaming/JsonLines.java

```diff
@@ -0,0 +1,395 @@
+package com.linecorp.armeria.server.streaming;
+
+import static com.linecorp.armeria.internal.server.ResponseConversionUtil.streamingFrom;
+import static java.util.Objects.requireNonNull;
+
+import java.io.ByteArrayOutputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.concurrent.Executor;
+import java.util.function.Function;
+import java.util.stream.Stream;
+
+import javax.annotation.Nullable;
+
+import org.reactivestreams.Publisher;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+
+import com.linecorp.armeria.common.HttpData;
+import com.linecorp.armeria.common.HttpHeaderNames;
+import com.linecorp.armeria.common.HttpHeaders;
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.common.HttpStatus;
+import com.linecorp.armeria.common.MediaType;
+import com.linecorp.armeria.common.ResponseHeaders;
+import com.linecorp.armeria.common.util.Exceptions;
+
+/**
+ * A utility class which helps to create a <a href="https://jsonlines.org/">JavaScript Object
+ * Notation (JSON) Lines text</a> from a content {@link Publisher} or {@link Stream}.
+ *
+ * <p>A user simply creates a streaming {@link HttpResponse} which emits JSON Lines text, e.g.
+ * <pre>{@code
+ * ObjectMapper mapper = new ObjectMapper().disable(SerializationFeature.INDENT_OUTPUT);
```

Comment:
How about using another `SerializationFeature` for a realistic example. As you know, users don't need to disable `INDENT_OUTPUT` manually.
```suggestion
 * ObjectMapper mapper = new ObjectMapper().enable(SerializationFeature.WRITE_ENUMS_USING_INDEX);
```

Reply:
done